homeassistant/core.py

```diff
@@ -1472,6 +1475,7 @@ def as_dict(self) -> Dict:
             "state": self.hass.state.value,
             "external_url": self.external_url,
             "internal_url": self.internal_url,
+            "media_dirs": self.media_dirs,
```

Comment:
Why would we make this available to the frontend?

Reply:
Is it frontend? As the method is just a `as_dict()` returning all values. `whitelist_external_dirs` and `allowlist_external_dirs` are in there too for example.

I would assume the part serving this to the frontend control what is shown, or we should rename the method. There is nothing in here that indicates this is for the frontend.

Reply:
We can rename that in another PR. For now let's not add it because it's not used.